packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesUtils.kt

```diff
@@ -256,9 +261,15 @@ object UpdatesUtils {
   }
 
   @Throws(ParseException::class)
-  fun parseDateString(dateString: String?): Date {
+  fun parseDateString(dateString: String?): Date? {
+    if (dateString == null) {
+      return null
+    }
```

Comment:
I think this is a somewhat risky change as it changes the defaulting of fields that are expected to be non-null. Instead, handle null at the callsites that pass nullable so that we don't need the `?: Date()` cases at all the callsites

```suggestion
  fun parseDateString(dateString: String): Date {
```